module/cmd-send/src/main/java/net/md_5/bungee/module/cmd/send/CommandSend.java

```diff
@@ -50,8 +50,12 @@ public void lastEntryDone()
                 ComponentBuilder builder = new ComponentBuilder( "" );
                 if ( !entry.getValue().isEmpty() )
```

Comment:
Shouldn't it be synchronised from here and also down below to when entry.getValue stops being used (ie put value in a local and synchronise it)

Reply:
I thought it only needs to be synchronized while iterating, that is only the case in the joiner call

Reply:
Well ordinarily getValue.isEmpty could change, but if this method is the last entry then that suggests no synchronisation is actually needed at all (although callback.count should be atomic?)